custom_components/citymind_water_meter/component/helpers/const.py

```diff
@@ -35,32 +34,31 @@
 ENDPOINT_PARAMETER_CURRENT_MONTH = "current_month"
 ENDPOINT_PARAMETER_ALERT_TYPE = "alert_type"
 
-ENDPOINT_CONSUMER_OLD = f"{API_URL_OLD}/consumer"
-ENDPOINT_CONSUMER_NEW = f"{API_URL_NEW}/consumer"
+ENDPOINT_CONSUMER = f"{API_URL}/consumer"
 
-ENDPOINT_CONSUMPTION_NEW = f"{API_URL_NEW}/consumption"
+ENDPOINT_CONSUMPTION = f"{API_URL}/consumption"
 
-ENDPOINT_MUNICIPALS_OLD = f"{API_URL_OLD}/municipals"
-ENDPOINT_MUNICIPALS_NEW = f"{API_URL_NEW}/municipality"
+ENDPOINT_MUNICIPALS_OLD = f"{API_URL}/municipals"
+ENDPOINT_MUNICIPALS_NEW = f"{API_URL}/municipality"
 
-ENDPOINT_LOGIN = f"{ENDPOINT_CONSUMER_OLD}/login"
+ENDPOINT_LOGIN = f"{ENDPOINT_CONSUMER}/login"
 
-ENDPOINT_ME = f"{ENDPOINT_CONSUMER_OLD}/me"
-ENDPOINT_METERS = f"{ENDPOINT_CONSUMER_NEW}/meters"
+ENDPOINT_ME = f"{ENDPOINT_CONSUMER}/me"
+ENDPOINT_METERS = f"{ENDPOINT_CONSUMER}/meters"
 ENDPOINT_UNITS = f"{ENDPOINT_MUNICIPALS_OLD}/h1/measurmentunits"
 ENDPOINT_CUSTOMER_SERVICE = f"{ENDPOINT_MUNICIPALS_OLD}/municipalCustomerService"
-ENDPOINT_ALERTS_FOR_SETTINGS = f"{ENDPOINT_CONSUMER_NEW}/alertsForSettings"
-
-ENDPOINT_LAST_READ = f"{ENDPOINT_CONSUMPTION_NEW}/last-read"
-ENDPOINT_CONSUMPTION_DAILY = f"{ENDPOINT_CONSUMPTION_NEW}/daily/lastbillingCycle/{{meter_id}}/{{yesterday}}/{{today}}"
-ENDPOINT_CONSUMPTION_MONTHLY = f"{ENDPOINT_CONSUMPTION_NEW}/monthly/{{meter_id}}/{{current_month}}/{{last_day_month}}"
-ENDPOINT_VACATIONS = f"{ENDPOINT_CONSUMER_OLD}/vacations"
-ENDPOINT_MY_ALERTS = f"{ENDPOINT_CONSUMER_NEW}/myalerts"
-ENDPOINT_MY_ALERTS_SETTINGS = f"{ENDPOINT_CONSUMER_OLD}/myalerts/settings"
+ENDPOINT_ALERTS_FOR_SETTINGS = f"{ENDPOINT_CONSUMER}/alertsForSettings"
+
+ENDPOINT_LAST_READ = f"{ENDPOINT_CONSUMPTION}/last-read"
+ENDPOINT_CONSUMPTION_DAILY = f"{ENDPOINT_CONSUMPTION}/daily/{{meter_id}}/{{yesterday}}/{{today}}"
+ENDPOINT_CONSUMPTION_MONTHLY = f"{ENDPOINT_CONSUMPTION}/monthly/{{meter_id}}/{{current_month}}/{{last_day_month}}"
```

Comment:
Endpoint is not related to the default consumption but v1.1 of that endpoint, should be:
```python
ENDPOINT_CONSUMPTION_MONTHLY = f"{API_URL}/v1.1/consumption/monthly/{{meter_id}}/{{current_month}}/{{last_day_month}}"
```

Reply:
the v1.1 api response is slightly different and will result error

without v1.1 the response is:
![image](https://github.com/maorcc/citymind_water_meter/assets/3001759/7fad67f2-23ec-407d-b04c-38734cf9d838)

and with the v1.1 the response is:
![image](https://github.com/maorcc/citymind_water_meter/assets/3001759/3b515e28-a377-4607-9517-0417c397f0df)

and there is error 
`2024-05-23 16:05:26.700 ERROR (MainThread) [custom_components.citymind_water_meter.component.managers.home_assistant] Failed to load extract consumption (consumption-monthly) details, Meter {'meterCount': 11547, 'meterSn': '', 'fullAddress': '}, Error: 'str' object has no attribute 'get', Line: 904`

Reply:
I checked in developer tools the request / response of the endpoint, it uses v1.1, i guess that's the right endpoint we need to use,
Based on past modifications they have conducted soon the non v1.1 endpoint will be removed

Reply:
@elad-bar 
I've added now a new commit with the v1.1 endpoint for the monthly consumption